### PR TITLE
chore: add map markers to cache

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

File: app/graphql/cache.ts [hunk 1]
```diff
@@ -51,6 +51,9 @@ export const createCache = () =>
         // https://www.apollographql.com/docs/react/caching/cache-configuration/#customizing-cache-ids
         keyFields: [],
       },
+      MapInfo: {
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply app/graphql/cache.ts:54]: ie: `MapMarker`

[the remark this answers is not in the document]
Comment (Extheoisah) [reply app/graphql/cache.ts:54]: Somehow I don't get the data in the cache when I use `MapMarker` but using `MapInfo` gives this result
Screenshot 2023-01-27 at 11 01 46
Screenshot 2023-01-27 at 11 02 03

[the remark this answers is not in the document]
Comment (nicolasburtey) [reply app/graphql/cache.ts:54]: what is the issue with the first way of caching it? I think you don't understand fully how caching works. you should read the apolloClient documentation on nesting, ie: https://www.apollographql.com/docs/react/caching/overview#how-is-data-stored

[the remark this answers is not in the document]
Comment (Extheoisah) [reply app/graphql/cache.ts]: Yeah, read the doc before making the PR. Turns out the I couldn't visualise the cache data cox of React native debugger. Had to do a clean restart of everything.
I've updated the PR

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**